app/controllers/mixins/vm_show_mixin.rb

```diff
@@ -167,6 +167,6 @@ def set_session_data
   end
 
   def breadcrumb_name(model)
-    ui_lookup_for_model(model || self.class.model.name).pluralize
+    ui_lookup(:models => model || self.class.model.name)
```

Comment:
@chessbyte : `git grep` for ` ui_lookup(:table` vs `ui_lookup(:tables`
Also see `ui_lookup` in `lib/vmdb/global_methods.rb`. 

The plural/singular form is selected by the key `:table` vs `:tables`. That is confusing and that is not introduced by this PR. That is all over the code. 

This PR only removes the `ui_lookup_*` methods that are used almost nowhere and that in turn call the `Dictionary.gettext`. Limiting the number of variants of these calls in the code and making further transformations simpler.